internal/bundlereader/charturl.go

```diff
@@ -49,6 +49,10 @@ func ChartVersion(location fleet.HelmOptions, a Auth) (string, error) {
 
 		r.Client = authCli
 
+		if a.BasicHTTP {
+			r.PlainHTTP = true
+		}
```

Comment:
How about `InsecureSkipVerify`? Is there a specific reason for not propagating it?

Reply:
Got it, thanks for clarifying!